### ISSUE TITLE
Add google analytics

### ISSUE THREAD

**Issue by dumptyd** [avinassh/cemetery.io#7]

**Comment by avinassh** [comment]
should we?

**Comment by dumptyd** [comment]
yes, we should. Do you want me to add it?

**Comment by avinassh** [comment]
No, I am thinking should we add a tracker or not. It isn't that much of a
high traffic site anyways no?

**Comment by frag-o-matic** [comment]
don't disrepect those that have passed. Please keep analytics away. Enough fuss with UIDAI as it is :pfftch
